File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Concepts/AspNetWrappers/C1BarChart/C1BarChartForm.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace ComponentOneSampleAppsCS
{
    public partial class C1BarChartForm : Form
    {
        public C1BarChartForm()
        {
            InitializeComponent();
            //Set default theme for wrapper
            mobjWrapper.Theme = "aristo";
        }

        /// <summary>
        /// Handles the Click event of the mobjSetButton control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjSetButton_Click(object sender, EventArgs e)
        {
            //Set Series Label to the new text value
            mobjWrapper.SeriesList[0].Label = mobjTextBox.Text;
            mobjWrapper.AspUpdate();
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Applications/ComponentOne/ComponentOneSampleApps/VB/C1BarChartForm.vb
Imports Gizmox.WebGUI.Forms
Imports Gizmox.WebGUI.Common
Imports Gizmox.WebGUI.Forms.Client

Namespace Gizmox.WebGUI.Forms.CompanionKit.ComponentOneSampleAppsVB

    Public Class C1BarChartForm

        Public Sub New()

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

            'Set default theme for wrapper
            mobjWrapper.Theme = "aristo"

        End Sub
        ''' <summary>
        ''' Handles the Click event of the mobjSetButton control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        Private Sub mobjSetButton_Click(sender As Object, e As EventArgs) Handles mobjSetButton.Click
            'Set Series Label to the new text value
            mobjWrapper.SeriesList(0).Label = mobjTextBox.Text
            mobjWrapper.AspUpdate()
        End Sub
    End Class

End Namespace